flink-runtime/src/main/java/org/apache/flink/runtime/memory/MemoryManager.java

```diff
@@ -229,7 +229,7 @@ public boolean isShutdown() {
 	}
 
 	/**
-	 * Checks if the memory manager all memory available.
+	 * Checks if the memory manager's all memory is available.
```

Comment:
"Checks if the memory manager's memory is completely available (nothing allocated at the moment)."

Reply:
Resolved. Thanks.